docs/resources/enable_data_protection.md

```diff
@@ -71,6 +79,14 @@ Required:
 - `provisioner_name` (String) Cluster provisioner name
 
 
+<a id="nestedblock--scope--cluster_group"></a>
+### Nested Schema for `scope.cluster_group`
+
+Required:
+
+- `cluster_group_name` (String) Cluster group name
```

Comment:
@shobha2626 - Why we have clustergroup as required here? It is supposed to be added later on right?

Reply:
Yes, it is supposed to be added after we claim DP support for cluster group scope.

But the schema change exist.

Reply:
Will be fixed in a separate PR by @shobha2626

Reply:
Fix is in PR - https://github.com/vmware/terraform-provider-tanzu-mission-control/pull/350